### ISSUE TITLE
Make tags version search pattern configurable

### ISSUE THREAD

**Issue by McFoggy** [jgitver/jgitver#20]
As a _jgitver user_ I would like the filtering pattern to discover tags acting as base version to be configurable so that it can handle different needs on different projects.

- __Must__: a recognition regexp pattern is configurable
- __Must__: the default is backward compatible

**Comment by randakar** [comment]
Nice! When will this get a release so we can try this?

**Comment by McFoggy** [comment]
it is already available. I have released this week-end jgitver 0.3.0 & jgitver-maven-plugin 0.4.0.

``` xml
  <extensions xmlns="http://maven.apache.org/EXTENSIONS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/EXTENSIONS/1.0.0 http://maven.apache.org/xsd/core-extensions-1.0.0.xsd">
    <extension>
      <groupId>fr.brouillard.oss</groupId>
      <artifactId>jgitver-maven-plugin</artifactId>
      <version>0.4.0</version>
    </extension>
  </extensions>
```

Have a look at the new [configuration](https://github.com/jgitver/jgitver-maven-plugin#configuration) regexVersionTag element. And from now you can start using the published XSD for configuration purpose.

I have started to publish them as a child of https://jgitver.github.io/ at https://jgitver.github.io/maven/configuration/index.html.

As always do not hesitate to comment, post issues and so on ; community help and feedback is very much appreciated.

**Comment by randakar** [comment]
Thanks. That's exactly what we needed. Somebody on our team will probably play with this during the current sprint.
